chimney/src/main/scala/io/scalaland/chimney/internal/macros/Model.scala

```diff
@@ -24,7 +24,10 @@ trait Model extends TransformerConfigSupport {
       }
   }
 
-  case class TransformerBodyTree(tree: Tree, isWrapped: Boolean)
+  type RenamesMapTree = c.Expr[Map[String, String]]
+  private val EmptyRenamesMapTree = c.Expr(q"${Map.empty[String, String]}")
+  case class TransformerTree(tree: Tree, renamesMapTree: RenamesMapTree = EmptyRenamesMapTree)
```

Comment:
I think that the difference between `TransformerTree` and `TransformerBodyTree` is subtle and confusing on the read site.